include/sbg_driver/sbg_matrix3.h

```diff
@@ -1,14 +1,55 @@
+/*!
+*	\file         sbg_matrix3.h
+*	\author       SBG Systems
+*	\date         13/03/2020
+*	 
+*	\brief        Handle a 3x3 matrix.
```

Comment:
Please improve your comments!
Such as:
Brief: Simple 3x3 matrix implementation
SBG Systems ROS driver need some basic vector and matrix operations.
ROS usually uses Eigen for all mathematical computations but to avoid a dependency on Eigen, it has been decided to implement just the needed basic mathematical operations in the driver.